android/src/main/java/ee/forgr/capacitor_updater/CapacitorUpdaterPlugin.java

```diff
@@ -163,7 +163,7 @@ public void notifyListeners(final String id, final JSObject res) {
     this.implementation.appId = config.getString("appId", "");
     this.implementation.appId =
       this.getConfig().getString("appId", this.implementation.appId);
-    if (!"".equals(implementation.appId)) {
+    if ("".equals(implementation.appId)) {
```

Comment:
Should this be `this.implementation.appId`?

What's the impact of this bug? Do we need to share this in the discord to get some eyes on it?

Reply:
nvm, I just saw #275